src/js/view/controls/tizen/tizen-controls.ts

```diff
@@ -85,20 +98,19 @@ class TizenControls extends Controls {
         // Pause Display
         if (!this.pauseDisplay) {
             const pauseDisplay = createElement(PauseDisplayTemplate());
-
             const title = new Title(model);
             title.setup(pauseDisplay.querySelector('.jw-pause-display-container'));
             this.div.appendChild(pauseDisplay);
             this.pauseDisplay = pauseDisplay;
         }
 
         // Display Buttons - Buffering
-        if (!this.displayContainer) {
-            const displayContainer = new DisplayContainer(model, api);
+        const displayContainer = new DisplayContainer(model, api);
```

Comment:
The controls interface can be enabled and disabled multiple times (`jwplayer().setControls()`). We do not want to redo work each time that controls are enabled, that only needs to happen the first time.